### ISSUE TITLE
Can workers deployed by helm form database sharding?

### ISSUE THREAD

**Issue by FriskKiddo** [manticoresoftware/manticoresearch-helm#81]
I deployed a manticore cluster with 8 workers. Can I use the cluster as the cluster with 8 shardings.

Well, for example, I create a rt table A in 8 workers, and then create distributed_table in 8 nodes with the configuration of agent(https://manual.manticoresearch.com/Creating_a_table/Creating_a_distributed_table/Creating_a_distributed_table). Each agent of the table in each nodes is set by other 7 workers. So I can query all the data in 8 workers via any distributed_table in workers.

Is it reasonable?

**Comment by sanikolaev** [comment]
Yes, it's reasonable. [Here's](https://github.com/manticoresoftware/manticoresearch-helm/issues/76) the issue about adding support for distributed tables. PRs are welcome. Manticore Search (dev version) can replicate distributed tables (which was a blocker for this). What's left is to make the helm chart use it.
